src/main/java/us/spaceclouds42/zones/mixin/ServerPlayerEntityMixin.java

```diff
@@ -52,18 +64,23 @@ public boolean isInBuilderMode() {
 
     @Override
     public boolean isBuilder() {
-        return BuilderManager.INSTANCE.isBuilder(((PlayerEntity) (Object) this).getUuid());
+        return BuilderManager.INSTANCE.isBuilder(this.getUuid());
     }
 
     @Override
     public void swapInventories() {
-        PlayerInventory tempInventory = new PlayerInventory((PlayerEntity) (Object) this);
+        PlayerInventory tempInventory = new PlayerInventory(this);
         tempInventory.clone(this.secondaryInventory);
         tempInventory.selectedSlot = this.currentInventory.selectedSlot;
         this.secondaryInventory.clone(this.currentInventory);
         this.currentInventory.clone(tempInventory);
 
         this.isInBuilderMode = !this.isInBuilderMode;
+
+
+        for (StatusEffectInstance effect : this.getStatusEffects()) {
```

Comment:
what's the purpose of this? afaik, when you enter/leave a zone, your status effects arent effected, why resend packets for them?